start_admin.sh

```diff
@@ -122,15 +166,15 @@ declare -i use_eic=0
 if [[ $authorized_keys_command == /opt/aws/bin/eic_run_authorized_keys* ]] \
 && [[ $authorized_keys_command_user == "ec2-instance-connect" ]]; then
   use_eic=1
-  ((++available_auth_methods))
+  ((++available_ssh_methods))
 fi
 
 chown -R "${LOCAL_USER}:" "${USER_SSH_DIR}"
 
-# If there were no successful auth methods, then users cannot authenticate
-if [[ "${available_auth_methods}" -eq 0 ]]; then
+# If there were no available SSH auth methods, then users cannot connect via the SSH daemon
+if [[ "${available_ssh_methods}" -eq 0 ]]; then
   user_data_condensed=$(jq -e -c . "${USER_DATA}" 2>/dev/null || cat "${USER_DATA}")
-  log "Failed to configure ssh authentication with user-data: ${user_data_condensed}"
+  log "No SSH authentication methods available in admin container user-data: ${user_data_condensed}"
 fi
```

Comment:
Do we still want to print out user data in the event that SSH authentication is not configured now that we're adding serial console access?